docs/releases/1.11.txt

```diff
@@ -292,6 +292,9 @@ Management Commands
 * The new :option:`diffsettings --default` option allows specifying a settings
   module other than Django's default settings to compare against.
 
+* The :option:`showmigrations --plan` now accepts a list of app labels
+  instead of ignoring them (:ticket:`27432`).
```

Comment:
No need for ticket reference, that's only for bugfixes in minor version releases.